.goreleaser.yml

```diff
@@ -43,12 +45,45 @@ archives:
       darwin: mac
 checksum:
   algorithm: sha512
+
+nfpms:
+  - id: default
+    package_name: caddy
+
+    # file_name_template: "{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}{{ if .Mips }}_{{ .Mips }}{{ end }}"
```

Comment:
Is this a TODO or something?

Reply:
Ah, no, just wanted to leave it in as a note that we can modify the filename template. I think we'll probably stick with the default because it matches the rest of our archives.